scripts/qa_crowbarsetup.sh

```diff
@@ -1747,7 +1753,6 @@ function reboot_nodes_via_ipmi
                 "default gateway to be active in bmc"
 
             $ipmicmd chassis bootdev pxe options=persistent
-            $ipmicmd mc reset warm
```

Comment:
This seems to be an unrelated change.

Why is the revert needed?
During our tests on the qa4 hardware we found that a consistent warm reset (reboot of BMC without factory reset) helps to bring up all BMCs with the wanted settings (they might have different settings from an earlier run).

If the warm reset really needs to be dropped, then the next two wait_for's also have to be removed as they make no more sense then.

Please also discuss this revert with @kbaikov , @vmoravec  and the qa4 hardware.